lib/flight_job/models/job.rb

```diff
@@ -346,17 +358,18 @@ def submit
       end
       script = load_script
 
-      FlightJob.logger.info("Submitting Job: #{id}")
-      cmd = [
-        FlightJob.config.submit_script_path,
-        script.script_path
-      ]
-
       # Generate the initial metadata path file
       FileUtils.mkdir_p File.dirname(initial_metadata_path)
       File.write initial_metadata_path, YAML.dump(active_metadata)
 
+      # Duplicate the script into the job's directory
+      # NOTE: Eventually this should probably be named after the job_name question
+      metadata["rendered_path"] = File.join(FlightJob.config.jobs_dir, id, script.script_name)
```

Comment:
We now have a `job_dir` method so this can be `File.join(job_dir, script.script_name)`.

Reply:
119deea06f38bd9857c38de081e30bf0a5f6ba46